bluesky/run_engine.py

```diff
@@ -936,16 +1013,25 @@ def request_suspend(self, fut, *, pre_plan=None, post_plan=None,
 
         """
         if not self.resumable:
+            # TODO fix race condition here!
```

Comment:
Flagging so we don't forget it. Is this an "open an issue" problem or a "fix before merging" problem?